Misc/NEWS.d/next/Core_and_Builtins/2024-10-09-13-53-50.gh-issue-125038.ffSLCz.rst

```diff
@@ -0,0 +1,3 @@
+Fix crash when iterating over a generator expression after direct changes on ``gi_frame.f_locals``.
+Tests on generator modifying through ``gi_frame.f_locals`` are added,
+both to genexpr generators and function generators. Patch by Mikhail Efimov.
```

Comment:
```suggestion
Fix crash when iterating over a generator expression after direct changes on ``gi_frame.f_locals``.
Patch by Mikhail Efimov.
```
We generally do not add notes about adding tests when fixing bugs, so we can omit that :)

Reply:
Thx!